simtools/model/camera.py

```diff
@@ -177,21 +176,13 @@ def _rotatePixels(self, pixels):
             The pixels dictionary with rotated pixels.
             The pixels orientation for plotting is added to the dictionary in pixels['orientation'].
             The orientation is determined by the pixel shape (see readPixelList for details).
-
-        Notes
-        -----
-        The additional rotation angle to get to the camera view of an observer facing the camera
-        is saved in the const dictionary CAMERA_ROTATE_ANGLE.
-        In the case of dual mirror telescopes, the axis is flipped in order to keep the same
-        axis definition as for single mirror telescopes.
-        One can check if the telescope is a two mirror one with isTwoMirrorTelescope.
         """
 
-        if isTwoMirrorTelescope(self._telescopeModelName):
-            pixels["y"] = [(-1) * yVal for yVal in pixels["y"]]
-
         rotateAngle = pixels["rotateAngle"]  # So not to change the original angle
-        rotateAngle += np.deg2rad(CAMERA_ROTATE_ANGLE[self._cameraName])
+        # The original pixel list is given such that x -> North, y -> West,
+        # i.e., x points down and y points to the right (for single-mirror telescopes).
```

Comment:
I don't understand the connection of North -> down and West -> right. This is from looking at a map?

Reply:
The full explanation is (excerpt from an e-mail from Konrad):

```
x -> North, y -> West, z -> Up in the ground system.
At azimuth=0, zenith angle=0 all coordinate systems are aligned.
When you turn the telescope the "normal" way towards
the horizon, the x-axis points downwards, the z-axis
points in any case from (primary) dish towards camera
```

I wanted to avoid writing so much text in the code so I shortened it, but I guess it is necessary to understand. So I will add this entire explanation (unless you think that even with that it is not clear).